src/app/sign-in/components/form-sign-in/form-sign-in.component.ts

```diff
@@ -109,14 +111,14 @@ export class FormSignInComponent implements OnInit, AfterViewInit {
       this.loading = true
 
       const isOauth = this.signInLocal.type === TypeSignIn.oauth
-      const $signIn = this._signIn.signIn(this.signInLocal, isOauth)
+      const $signIn = this._signIn.signIn(this.signInLocal, isOauth, isOauth)
```

Comment:
I don't think this is right the parameters should be `updateUserSession` and `forceSessionUpdate` instead of isOauth

Reply:
This is right (at the moment). Since only on Oauth mode logins we want to update the session, in force update mode. Here is more info about it https://github.com/ORCID/orcid-angular/blob/8abd9d6121dbbada43f5aafe54402d360ea61c6a/src/app/core/sign-in/sign-in.service.ts#L33-L34

Reply:
well in fact `forceSessionUpdate` is only needed when is an Oauth authorization for an Open id with a `forced login`. But it does not make any harm to use it for all oauts logins.